src/main/java/frc/robot/subsystems/VisionSubsystem.java

```diff
@@ -40,61 +36,72 @@
  * means that from the (red) driver's perspective +X is away and +Y is to the right.
  */
 public class VisionSubsystem extends SubsystemBase {
-  private static final double CAMERA_X_POS_METERS = Units.inchesToMeters(27.0 / 2.0 - 0.94996);
+  // don't mind the value is never used part it is used
+  private static final double CAMERA_X_POS_METERS_FRONT =
```

Comment:
You should include in the above comment info on what the math is below, so that future maintainers know what each value in the equation is for, so that they can adjust it properly if needed.